### PR TITLE
dev-go/twofactor: version bump to 0_p20180418

### PR DESCRIPTION
**Pull request by Tatsh**
*dev-go/twofactor*: Bump to *20180418* commit which fixes issue reading unpadded base32 TOTP secrets.

## Related

https://github.com/gokyle/twofactor/issues/10
https://github.com/browserpass/browserpass/issues/255

### CODE REVIEW

File: dev-go/twofactor/Manifest [hunk 1]
```diff
@@ -1 +1 @@
-DIST twofactor-0_p20170917.tar.gz 6637 BLAKE2B e6090a420f7d98063aa01f1c88c1dbe20103cdc4fa30d8fa9b235431f2d8cdcf1aee3cf7773a97916716aa8d0e257adc3f854db138da7c2541eef7cf1ebd493b SHA512 d3cd43d7fe2c819fa9a4ea1eacdb9508d20d54d638239c379fa8ca6ca83ecd67a06aff054c023b55e4555a227a725fd21caace55159a544fd389e30e67c363ad
+DIST twofactor-0_p20180418.tar.gz 22515 BLAKE2B 460098bf0ff570a896eb9b596ecee05ecf35219aa6ad8a5b2e8763740f588c158d186941e873623c291694eb35a2390feb372e04adf1a667aed5f884fb0b436d SHA512 ed02ab278f753d05f34b8a1e1656174cf36ce7526501d06056bdf66aaa714515a4f585b0216f987fa1cc8192b0ecfd685c729bbdb25ffe7522ba7cb8076ff026
```
Comment (Zlogene) [inline dev-go/twofactor/Manifest]: removal and addition should go in two different commits

Comment (Tatsh) [reply dev-go/twofactor/Manifest]: Resolved with b7ec54b1d888ae02834d36b99194aa2eb693368a

**Comment by Tatsh** [comment]
Version 2.0.22 of browserpass is already in the tree, so this now just updates *dev-go/twofactor* for the unpadded base32 TOTP secrets.

**Comment by mgorny** [comment]
Thank you. It looks good and passes tests. I'll merge it now.
